tex/pgfcircmultipoles.tex

```diff
@@ -2160,4 +2160,53 @@
 }
 % %>>>
 
+%% definitions and shape for euroinv %<<<
+\ctikzset{european inversion mark/width/.initial = 0.2}
+\ctikzset{european inversion mark/height/.initial = 0.1}
+\pgfdeclareshape{euroinv}{
+    \savedmacro{\ctikzclass}{\edef\ctikzclass{muxdemuxes}}
+    \saveddimen{\scaledRlen}{\pgfmathsetlength{\pgf@x}{\ctikzvalof{\ctikzclass/scale}\pgf@circ@Rlen}}
+    \savedanchor\northeast{%
+        \pgfmathsetlength{\pgf@circ@scaled@Rlen}{\ctikzvalof{\ctikzclass/scale}\pgf@circ@Rlen}
+        \pgf@y=\ctikzvalof{european inversion mark/height}\pgf@circ@scaled@Rlen
+        \pgf@x=\ctikzvalof{european inversion mark/width}\pgf@circ@scaled@Rlen
```

Comment:
I think the default size is somewhat small and has to be increased... i am testing on it to see if it has to do with environment it's used in, but this is how it's looking on my document compared with the dimensions i previously used.

![image](https://user-images.githubusercontent.com/19423063/215324313-00f4b741-d878-4f9c-b206-26925708728a.png)

![image](https://user-images.githubusercontent.com/19423063/215325706-021ed688-c572-44ac-88dd-35f8be69d111.png)

![image](https://user-images.githubusercontent.com/19423063/215326333-3dfab25c-b45f-4b48-a8a5-e9f921109848.png)

Reply:
Maybe
```
\ctikzset{european inversion mark/width/.initial = 0.3}
\ctikzset{european inversion mark/height/.initial = 0.15}
```
will do? (I don't want to change the lines you marked there --- I like the fact that the units are the same as `external pins` etc...)

Reply:
> I don't want to change the lines you marked there...

yeah it's okay, i just semi-randomly selected these lines... was not saying particularly... 
as i don't have much experience with pgf layer of code

Reply:
yeah, `0.3,0.15` is working good.... 

i keep getting hung up in editors, aaahh... that's why it takes too much time on my part

![image](https://user-images.githubusercontent.com/19423063/215334506-e9333bd7-a730-4bc2-aff8-8d27d8b98a99.png)

Reply:
yeah, i have thought about smth similar for the tutorial. i am just clearing the backlog of issues from so long... i will soon get to writing the tutorial for #665